__init__.py

```diff
@@ -0,0 +1,3 @@
+from app import app
+
+app.config.from_pyfile('config.py')
```

Comment:
i don't think this file is needed , you can import this config at the point of initialization of your app.